firmware/config/boards/subaru_eg33/openblt/led.c

```diff
@@ -0,0 +1,100 @@
+/************************************************************************************//**
+* \file         Demo/ARMCM7_STM32F7_Nucleo_F767ZI_GCC/Boot/led.c
+* \brief        LED driver source file.
```

Comment:
please move out of boards/ into hw_layer/stm32/f7 for other boards to reuse
please PR all openblt files in a separate PR and integration with rusEFI as another one

Reply:
Compromise solution: reuse HAL and CMSIS files from OpenBLT Demo project. As we do not heed this files for RE itself there is no reason to have it RE git.
Only two additional files have to be added for HAL configuration: config header and system.c file with CPU configuration.